docs/client/full-api/api/pubsub.md

```diff
@@ -173,10 +173,11 @@ collection name.
 The client will see a document if the document is currently in the published
 record set of any of its subscriptions.
 
-The `onReady` callback is called with no arguments when the server
-[marks the subscription as ready](#publish_ready). The `onError` callback is
-called with a [`Meteor.Error`](#meteor_error) if the subscription fails or is
-terminated by the server.
+The `onReady` callback is called with no arguments when the server [marks the
+subscription as ready](#publish_ready). The `onStopped` callback is called with
+a [`Meteor.Error`](#meteor_error) if the subscription fails or is terminated by
+the server. If the subscription is stopped for any other reason, `onStopped` is
```

Comment:
Ah yes I guess I meant --- What you are doing here is totally the way we do things here.  I'm starting to think it's not ideal for users, though.

It wouldn't actually be all that hard to upload all the old docs apps...